core/services/standardcapabilities/delegate.go

```diff
@@ -91,7 +96,8 @@ func NewDelegate(
 		isNewlyCreatedJob:       false,
 		gatewayConnectorWrapper: gatewayConnectorWrapper,
 		ks:                      ks,
-		peerWrapper:             peerWrapper,
+		externalPeerWrapper:     externalPeerWrapper,
+		ocrPeerWrapper:          ocrPeerWrapper,
```

Comment:
@vreff Why do you need the ocrPeerWrapper here? It should be enough to use the externalPeerWrapper 🤔

Reply:
I don't, but the OCR peer wrapper was just called "peerWrapper" so I've renamed it to add the distinction between the wrappers.

Reply:
My question was more -- why are we adding the dependency? It didn't used to be there, so why do you need it now 🤔 ?